runtime/doc/lsp.txt

```diff
@@ -61,6 +61,16 @@ options are not restored when the LSP client is stopped or detached.
 - |K| is mapped to |vim.lsp.buf.hover()| unless |'keywordprg'| is customized or
   a custom keymap for `K` exists.
 
+                                                *gll* *glr* *gln* *i_CTRL-S*
```

Comment:
```suggestion
                                                *grl* *grr* *grn* *i_CTRL-S*
```

Reply:
Oops, I actually _did_ mean for the mappings to be `gl` prefixed, not `gr`, but I forgot to change them in `_defaults.lua`.